node/tessel-export.js

```diff
@@ -1006,11 +1017,15 @@ Tessel.SPI = function(params, port) {
 };
 
 Tessel.SPI.prototype.send = function(data, callback) {
-  this._port.cork();
+
   this.chipSelect.low();
+  this._port.delay(this.chipSelectDelayUs);
```

Comment:
Wrap both of these in a condition, because if the delay is zero, we definitely do not want to waste the 3ms sending a message that effectively means "don't do anything"